easybuild/easyconfigs/a/awscli/awscli-2.1.37-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,70 @@
+easyblock = 'PythonBundle'
+
+name = 'awscli'
+version = '2.1.37'
+
+homepage = 'https://pypi.python.org/pypi/awscli'
+description = 'Universal Command Line Environment for AWS'
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('PyYAML', '5.3.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('jmespath', '0.10.0', {
+        'checksums': ['b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9'],
+    }),
+    ('botocore', '1.20.53', {
+        'source_tmpl': '2c5e5782c4.tar.gz',
+        'source_urls': ['https://github.com/boto/botocore/archive/'],
+        'checksums': ['a8d6340ded75ce69bc10db9c5b37d0ef443fbad417a366c6a702b4a3d4594ad4'],
+    }),
+    ('docutils', '0.15.2', {
+        'checksums': ['a2aeea129088da402665e92e0b25b04b073c04b2dce4ab65caaa38b7ce2e1a99'],
+    }),
+    ('s3transfer', '0.3.7', {
+        'checksums': ['35627b86af8ff97e7ac27975fe0a98a312814b46c6333d8a6b889627bcd80994'],
+    }),
+    ('ruamel.yaml', '0.15.100', {
+        'checksums': ['8e42f3067a59e819935a2926e247170ed93c8f0b2ab64526f888e026854db2e4'],
+    }),
+    ('prompt_toolkit', '2.0.10', {
+        'checksums': ['f15af68f66e664eaa559d4ac8a928111eebd5feda0c11738b5998045224829db'],
+    }),
+    ('cryptography', '3.3.2', {
+        'checksums': ['5a60d3780149e13b7a6ff7ad6526b38846354d11a15e21068e57073e29e19bed'],
+    }),
+    ('distro', '1.5.0', {
+        'checksums': ['0e58756ae38fbd8fc3020d54badb8eae17c5b9dcbed388b17bb55b8a5928df92'],
+    }),
+    ('wcwidth', '0.1.9', {
```

Comment:
0.2.5 is already included in Python 3.8.6, do you need to downgrade it?

Reply:
```
requires = [
    'botocore==2.0.0dev105',
    'colorama>=0.2.5,<0.4.4',
    'docutils>=0.10,<0.16',
    'cryptography>=3.3.2,<3.4.0',
    's3transfer>=0.3.0,<0.4.0',
    'ruamel.yaml>=0.15.0,<0.16.0',
    # wcwidth 0.2.0 and up do not get copied into
    # the pyinstaller executable.
    'wcwidth<0.2.0',
    'prompt-toolkit>=2.0.0,<3.0.0',
    'distro>=1.5.0,<1.6.0',
]
```

Reply:
Yes, awscli has very specific requirements.  In cases like this should a patch be made to change setup.py to make versions more liberal?